app/models/server/models/LivechatExternalMessages.js

```diff
@@ -0,0 +1,16 @@
+import { Base } from './_Base';
+
+export class LivechatExternalMessage extends Base {
+	constructor() {
+		super('livechat_external_message');
```

Comment:
What about creating an index for the `rid` field?
It would provide more performance when finding records.